### PR TITLE
use package.json scripts instead of Makefile

### PR DESCRIPTION
**Pull request by pierreinglebert**

### CODE REVIEW

**Comment by XhmikosR** [comment]
Oops, didn't see that :/

File: package.json [hunk 1]
```diff
@@ -52,8 +52,13 @@
     "zopflipng": "bin/zopflipng"
   },
   "scripts": {
+    "build-source": "node-pre-gyp rebuild",
+    "clean": "rm -rf ./lib/binding; rm -rf ./coverage",
```
Comment (XhmikosR) [inline package.json]: I'd say just remove this because it's not cross-platform.

**Comment by pierreinglebert** [comment]
Yep, we can merge this, can you take the travis and coveralls parts ?

**Comment by XhmikosR** [comment]
I don't quite follow...

**Comment by pierreinglebert** [comment]
I made this cause I'd like to merge the appveyor pr and publish a new version today :)

**Comment by pierreinglebert** [comment]
Take my travis and coveralls modifications to your pr and I'll close it.

**Comment by XhmikosR** [comment]
Oh, OK, let me try.

**Pull request closed without merging**